Source/carthage/Archive.swift

```diff
@@ -14,19 +14,21 @@ import ReactiveCocoa
 
 public struct ArchiveCommand: CommandType {
 	public struct Options: OptionsType {
+		public let toolchain: String?
 		public let outputPath: String?
 		public let directoryPath: String
 		public let colorOptions: ColorOptions
 		public let frameworkNames: [String]
 
-		static func create(outputPath: String?) -> String -> ColorOptions -> [String] -> Options {
-			return { directoryPath in { colorOptions in { frameworkNames in
-				return self.init(outputPath: outputPath, directoryPath: directoryPath, colorOptions: colorOptions, frameworkNames: frameworkNames)
-			} } }
+		static func create(outputPath: String?) -> String? -> String -> ColorOptions -> [String] -> Options {
+			return { toolchain in { directoryPath in { colorOptions in { frameworkNames in
+				return self.init(toolchain: toolchain, outputPath: outputPath, directoryPath: directoryPath, colorOptions: colorOptions, frameworkNames: frameworkNames)
+			} } } }
 		}
 
 		public static func evaluate(m: CommandMode) -> Result<Options, CommandantError<CarthageError>> {
 			return create
+				<*> m <| Option<String?>(key: "toolchain", defaultValue: nil, usage: "the toolchain to build with")
```

Comment:
`carthage archive` doesn't build anything, so it shouldn't need the `toolchain` option.